clients/browser/style.css

```diff
@@ -37,29 +48,102 @@ body {
 }
 
 header {
-    width: 250px;
+    width: 100%;
     max-width: 100%;
     margin: auto;
     margin-bottom: 25px;
     text-align: center;
 }
 
 header .name {
+      color:#FFF;
 	line-height: 85px;
-	font-size: 70px;
+	font-size: 62px;
 	font-weight: lighter;
 	font-family: sans-serif;
 }
 
 header .name span {
 	color: #FFC107;
 	margin-right: 10px;
-	font-size: 54px;
+	font-size: 42px;
 }
 header .subtitle {
+    color:#FFF;
     font-size: 15px;
     letter-spacing: 2px;
 }
+header a.headerlink {
+    float:right;
```

Comment:
Leave a space between the CSS attribute and the value. Same holds for the whole file.